InternTrack.Core.Api.Infrastructure.Build/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using ADotNet.Clients;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks.SetupDotNetTaskV1s;
+
+var githubPipeline = new GithubPipeline
```

Comment:
Where's the namespace? scope? classname?

Reply:
@hassanhabib In the Sofee video for this section you did'nt put in a namespace, scope, or classname in program.cs. Do you want me to show him how to do it?

Reply:
@Pwongchaiya yup show him.